include/dao/dao.js

```diff
@@ -114,6 +117,48 @@ module.exports = function DAOModule(pb) {
         this.loadByValues(where, collection, opts, cb);
     };
 
+    DAO.prototype.loadByValueAvailableToSite = function(key, val, site, collection, opts, cb) {
+        var self = this;
+        this.loadByValueForOneSite(key, val, site, collection, opts, function(err, result) {
+            if (util.isError(err)) {
+                return cb(err);
+            }
+
+            //ensure setting exists
+            if (!result){
+                self.loadByValueFromGlobal(key, val, collection, opts, cb);
+                return;
+            } else {
+                cb(null, result);
+            }
+        });
+    };
+
+    DAO.prototype.loadByValueForOneSite = function(key, val, site, collection, opts, cb) {
+        if(!site || site === GLOBAL_SITE) {
+            this.loadByValueFromGlobal(key,val,collection,opts,cb);
+        } else {
+            var where = {};
+            where[key] = val;
+            where[SITE_FIELD] = site;
+            this.loadByValues(where, collection, opts, cb);  
+        }
+    };
+
+    DAO.prototype.loadByValueFromGlobal = function(key, val, collection, opts, cb) {
```

Comment:
All low level functions should be documented.  It is necessary for our understanding of code intentions as well as providing meaningful documentation in our auto-generated documentation.